web_tool_script_3.R

```diff
@@ -45,8 +45,22 @@ scenario_other <- "ETP2017_B2DS"
 scenario_shipping <- "SBTI_SBTI"
 portfolio_allocation_method <- "portfolio_weight"
 scenario_geography <- "Global"
-audit_file <- read_csv(file.path(proc_input_path, portfolio_name_ref_all, "audit_file.csv"), col_types = cols())
-emissions <- read_rds(file.path(proc_input_path, portfolio_name_ref_all, "emissions.rda"))
+
+if (file.exists(file.path(proc_input_path, portfolio_name_ref_all, "audit_file.csv"))){
+  audit_file <- read_csv(file.path(proc_input_path, portfolio_name_ref_all, "audit_file.csv"), col_types = cols())
+}else{
+  audit_file <- tibble("investor_name" = NA_character_, "portfolio_name" = NA_character_,
+                       "asset_type" = NA_character_, "valid_input" = NA, "isin" = NA_character_,
+                       "direct_holding" = NA, "value_usd" = NA_real_)
```

Comment:
probably safest to use an explicitly typed `NA` value for `valid_input` and `direct_holding`